tests/queries/0_stateless/03148_mutations_virtual_columns.sql

```diff
@@ -0,0 +1,15 @@
+DROP TABLE IF EXISTS t_mut_virtuals;
+
+CREATE TABLE t_mut_virtuals (id UInt64, s String) ENGINE = MergeTree ORDER BY id;
+
+INSERT INTO t_mut_virtuals VALUES (1, 'a');
+INSERT INTO t_mut_virtuals VALUES (2, 'b');
+
+SET mutations_sync = 2;
+
+ALTER TABLE t_mut_virtuals UPDATE s = _part WHERE 1;
+ALTER TABLE t_mut_virtuals DELETE WHERE _part LIKE 'all_1_1_0%';
```

Comment:
The part name might not be constant in the cloud (because of keeper faults). Not sure if we need this explicitly, or we want to just disable ZK fault injection just in case.

Reply:
I think plain `MergeTree` is not affected by this issue.

Reply:
Yes, but this test will be executed for SharedMergeTree in the cloud too

Reply:
Ok, I think it's better to disable zk fault injections here.